packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/sqm-user-info-form.feature

```diff
@@ -0,0 +1,147 @@
+@owner:andy @author:andy
+Feature: Tax Form Step One
+
+  Background: A user is prompted to enter their personal and initial tax information
+    Given a user is on Tax Form Step One
+
+  @minutia
+  Scenario: The Participant is a partner and cannot change their email
+    When they are on the form
+    And they are already a partner
+    Then the email field is pre-filled with their email
+    And the email field is disabled
+
+  @minutia
+  Scenario: A user is filling out the form and selects their country
+    When they open the country dropdown
+    Then the available countries are provided:
+      | country        |
+      | UNITED STATES  |
+      | UK             |
+      | AUSTRALIA      |
+      | NEWZEALAND     |
+      | CANADA         |
+      | AUSTRIA        |
+      | BELGIUM        |
+      | BULGARIA       |
+      | CROATIA        |
+      | CYPRUS         |
+      | CZECHREPUBLIC  |
+      | DENMARK        |
+      | ESTONIA        |
+      | FINLAND        |
+      | FRANCE         |
+      | GERMANY        |
+      | GREECE         |
+      | HUNGARY        |
+      | IRELAND        |
+      | ITALY          |
+      | LATVIA         |
+      | LITHUANIA      |
+      | LUXEMBOURG     |
+      | MALTA          |
+      | NETHERLANDS    |
+      | POLAND         |
+      | PORTUGAL       |
+      | ROMANIA        |
+      | SLOVAKIA       |
+      | SLOVENIA       |
+      | SPAIN          |
+      | SWEDEN         |
+      | ICELAND        |
+      | INDIA          |
+      | ISRAEL         |
+      | JAPAN          |
+      | MEXICO         |
+      | NORWAY         |
+      | SINGAPORE      |
+      | SOUTHAFRICA    |
+      | SOUTHKOREA     |
+      | SWITZERLAND    |
+      | TAIWAN         |
+      | THAILAND       |
+      | PHILIPPINES    |
+      | MALAYSIA       |
+      | UAE            |
+      | TURKEY         |
+      | RUSSIA         |
+      | CANARY_ISLANDS |
+
+  @minutia
+  Scenario: A user is filling out the form and selects their currency
```

Comment:
Flagging for @Locrian24 that this is going to become dynamic based on country - just waiting on the API